examples/visualize_attractors.py

```diff
@@ -9,8 +9,10 @@
 import numpy as np
 import argparse
 import os
+import pdb
 
 if __name__ == "__main__":
+    print(os.getcwd() )
```

Comment:
Remove `print` statements and `import pdb` from this file since we may want to run it in a batch process on a large cluster.